sensor_msgs/CameraInfo  camera_info
geometry_msgs/Pose      robot_pose  # transformation from world to robot root frame
geometry_msgs/Pose      camera_pose # transformation from world to camera frame
sensor_msgs/JointState  state
---
sensor_msgs/Image       mask        # labels (0: background, >0: parts)
sensor_msgs/Image       points      # points in camera frame
string[]                link_names
geometry_msgs/Pose[]    link_poses  # link poses in camera frame
